### ISSUE TITLE
Dead link

### ISSUE THREAD

**Issue by oxoocoffee** [start-jsk/qnx_ros#4]
Looks like you have dead link in your README

http://www.ros.org/wiki/groovy/Installation/QNX

**Comment by k-okada** [comment]
yeah, Is http://wiki.ros.org/groovy/Installation/QNX correct one? Unfortunately ROS/QNX is not completed yet, so if you're interesting, we could start over this project again.

**Comment by oxoocoffee** [comment]
I think collaboration would be great. But here are my thoughts. Since ROS 2.0 is around the corrner would you be interested in making 2.0 work? on QNX 7? I am researching this myself right now so not 100% sure myself.
What are you thoughts?

**Comment by k-okada** [comment]
that's could be nice, I personally go into ROS2 because we have so many
ROS1 software/applications at this moment, but ROS2 and realtime OS could
be best match.

--
◉ Kei Okada


2017-11-14 5:28 GMT+09:00 Robert J. Gebis <notifications@github.com>:

**Comment by oxoocoffee** [comment]
I am waiting for my eval board. I hope to get it in couple of weeks is what I was told.
Are you on Slack?

**Comment by k-okada** [comment]
> Are you on Slack?

yes, not so responsive though.

--
◉ Kei Okada


2017-11-18 1:03 GMT+09:00 Robert J. Gebis <notifications@github.com>:
